tools/cvm-image-rewriter/plugins/06-install-tdx-guest-kernel/README.md

```diff
@@ -0,0 +1,22 @@
+# Install TDX guest kernel
+
+This plugin is used to install TDX guest kernel from a given local repository.
+
+# Prerequisite
+
+Prepare the local repository and confirm that there are Debian packages related to the TDX kernel in the `/jammy/amd64/` directory of this repository. You can choose to place this local repository in the `pre-stage/artifacts/` directory.
```

Comment:
Maybe instead of  "you can choose to place" to make it  "It is recommended to place"

Reply:
Or is it required to be here?  If it is required to be in this spot for this to work I would use stronger language telling users it is required.

Reply:
It's not required，recommended is fine. Thanks.